### PR TITLE
Use npm ci for consistent package versions.

### PR DESCRIPTION
**Pull request by AntonNeld**
Also specify Node version to use. This helps avoid hard-to-reproduce bugs.

### CODE REVIEW

**Comment by AntonNeld** [comment]
The node_modules directory is the equivalent of Python's venv (if I understand venv correctly), and everything in package.json will be installed there by default. This will be removed and recreated when you run 'npm ci'. The command will only fail if package-lock.json doesn't match package.json. Since both files are checked in, the only way for this to happen is if you modify one of them yourself.

When you install a new package, you would run 'npm install <package>', which will update both files. (Or you can add it to package.json and run 'npm install' to update package-lock.json. Then you might update some other packages as well, but that should be fine.)

When a new package is added everyone needs to run 'npm ci' (or without this change, 'npm install') when pulling. It should be possible to write a git hook to do this automatically, but I am not sure how to make it work cross-platform.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ Repository for the UI of Screencrash (also known as Skärmkrock)
 
 
 ## Dependencies
-The primary dependency for this repository is [Node.js](https://nodejs.org/) which needs to be installed and accessible.
+The primary dependency for this repository is [Node.js](https://nodejs.org/) 10.16.0 which needs to be installed and accessible.
+To install a specific version, you can use [NVM](https://github.com/nvm-sh/nvm).
```
Comment (AntonNeld) [inline README.md]: I realized this is only for Mac and Linux. I found [this](https://github.com/coreybutler/nvm-windows) for Windows but cannot vouch for it. Should I add it anyway?

**Comment by ITNano** [comment]
Ah, mixed up the projects so got kind of confused there (with python stuffs). Everything looks in order. I will approve.

Maybe we should add the git hook as a task so we don't forget it?

File: README.md (hunk 1 above)
Comment (ITNano) [reply README.md]: You could add a link to https://nodejs.org/dist/v10.16.0/ for Windows users. All distributables can be found there.

Comment (AntonNeld) [reply README.md]: How easy is it to change the Node version on Windows if we decide we need some new features or a package that depends on a later version? I just know it's a pain on Linux which is why NVM is so useful.

Comment (ITNano) [reply README.md]: Haven't done it but it's probably just installing a new executable and changing the PATH environment variable so it points to the correct path. The installer might even help you fix the PATH variable if it's kind enough. Shouldn't be a problem.

**Comment by AntonNeld** [comment]
> Maybe we should add the git hook as a task so we don't forget it?

That sounds like a good idea, should it be an issue or a note in the project?

**Pull request merged**